server/src/Tests.cpp

```diff
@@ -971,13 +971,29 @@ void KTestObjectParser::processSymbolicStdin(Tests::TestCaseDescription &testCas
 
 void KTestObjectParser::processSymbolicFiles(Tests::TestCaseDescription &testCaseDescription,
                                              const std::vector<RawKleeParam> &rawKleeParams) {
-    std::vector<Tests::TestCaseParamValue> filesValues(types::Type::symFilesCount);
+    std::vector<Tests::FileInfo> filesValues(types::Type::symFilesCount);
     for (char fileName = 'A'; fileName < 'A' + types::Type::symFilesCount; fileName++) {
+        std::string readBytesName = PrinterUtils::getFileReadBytesParamKTestJSON(fileName);
+        auto &&readBytes = getKleeParamOrThrow(rawKleeParams, readBytesName);
+        filesValues[fileName - 'A'].readBytes = std::stoi(
```

Comment:
always use `fileName - 'A'` looks very bad. Change fileValues to map or create functions fileNameToind and indToFileName.